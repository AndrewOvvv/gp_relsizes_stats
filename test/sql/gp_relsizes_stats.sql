CREATE EXTENSION gp_relsizes_stats;

SELECT always_return_one();

SELECT always_return_square(-2);
