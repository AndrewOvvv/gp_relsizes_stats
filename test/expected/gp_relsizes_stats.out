CREATE EXTENSION gp_relsizes_stats;
SELECT always_return_one();
 always_return_one 
-------------------
 (1)
(1 row)

SELECT always_return_square(-2);
 always_return_square 
----------------------
                    4
(1 row)

